pkgs/top-level/python-aliases.nix

```diff
@@ -6,9 +6,9 @@ let
   # Removing recurseForDerivation prevents derivations of aliased attribute
   # set to appear while listing all the packages available.
   removeRecurseForDerivations = alias: with lib;
```

Comment:
This file is a direct copy of aliases.nix and should be kept in sync. Please undo the formatting changes and only keep the added alias.

Reply:
fomatting undone